### PR TITLE
fix: API Tokens Table Style Regression

### PR DESCRIPTION
**Pull request by bnussman-akamai**
## Description 📝

Fixes a small style regression caused by design tokens 🎨 on the API tokens page

## Preview 📷

| Before  | After   |
| ------- | ------- |
| Screenshot 2024-09-11 at 11 02 52 AM | Screenshot 2024-09-11 at 11 02 58 AM |

## How to test 🧪

- Go to http://localhost:3000/profile/tokens
- Check styles on this page 👀 
  - Test in light 🌞  and dark 🌑  mode
  - Test on small screen sizes 📱 
  
## As an Author I have considered 🤔

- [ ] 👀 Doing a self review
- [ ] ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- [ ] 🤏 Splitting feature into small PRs
- [x] ➕ Adding a [changeset](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md#writing-a-changeset)
- [ ] 🧪 Providing/Improving test coverage
- [ ] 🔐 Removing all sensitive information from the code and PR description
- [ ] 🚩 Using a feature flag to protect the release
- [ ] 👣 Providing comprehensive reproduction steps
- [ ] 📑 Providing or updating our documentation
- [ ] 🕛 Scheduling a pair reviewing session
- [ ] 📱 Providing mobile support
- [ ] ♿  Providing accessibility support

### CODE REVIEW

**Comment by harsh-akamai** [comment]
Could we also add padding to the table to ensure it matches the styling of the other instances?

<details>
<summary>Reference</summary>

image

</details>

File: packages/manager/src/features/Profile/APITokens/APITokenTable.tsx [hunk 1]
```diff
@@ -185,32 +181,34 @@ export const APITokenTable = (props: Props) => {
...
+              isProxyUser ? PROXY_USER_RESTRICTED_TOOLTIP_TEXT : undefined
+            }
+            buttonType="primary"
+            disabled={isProxyUser}
+            onClick={() => setIsCreateOpen(true)}
+          >
+            Create a Personal Access Token
+          </Button>
+        )}
+      </Paper>
```
Comment (harsh-akamai) [inline packages/manager/src/features/Profile/APITokens/APITokenTable.tsx:211]: Could we also add padding to the table to ensure it matches the styling of the other instances?

<details>
<summary>Reference</summary>

image
</details>

Comment (bnussman-akamai) [reply packages/manager/src/features/Profile/APITokens/APITokenTable.tsx:211]: We could consider that, but I think this is a different design pattern used here. It is also used in other places like Linode Details 

Screenshot 2024-09-12 at 9 01 39 AM

Comment (harsh-akamai) [reply packages/manager/src/features/Profile/APITokens/APITokenTable.tsx:211]: Gotcha!

**Review by harsh-akamai — APPROVED** [review]

**Review by coliu-akamai — APPROVED** [review]
Confirmed style regression is gone (light/dark mode, small sizes) 🎉 

One small thing I noticed after you showed the Linode storage tab - seems like the the table titles are closer to the left table border in API tokens than they are in the Linode storage tab?

image
image

**Comment by bnussman-akamai** [comment]
Good catch @coliu-akamai 

I updated the padding to match

**Pull request merged**